app/models/manageiq/providers/microsoft/inventory/parser/infra_manager.rb

```diff
@@ -332,6 +333,18 @@ def parse_vm_hardware(vm, data)
     parse_vm_networks(hardware, data)
   end
 
+  def parse_vm_custom_properties(vm, data)
+    data['CustomProperty'].map do |key, value|
```

Comment:
For graph refresh you need to add this to the inventory collection:
```
persister.ems_custom_attributes.build(
  :resource => vm,
  :section  => "custom_field",
  :name     => key,
  :value    => value,
  :source   => 'VC'
)
```

Reply:
I have installed newer version of CFME 5.10.5.1, where is support of graph refresh for SCVMM provider, but I cannot see any ems custom attributes.

In case of use an old method - refresh_parser.rb it's working, but problem is in case any custom property in SCVMM is removed, it will stay in CFME as VM object

Reply:
@vasekbo I don't think that 5.10.5.1 has support for scvmm graph refresh actually.